docs/admin-guide/loading.md

```diff
@@ -86,6 +86,32 @@ human_genome_build: 37
 
 ```
 
+#### Adding custom images to a case
+
+Scout can display custom images as new panels on the case view which could be used to display analysis results from a seperate pipeline. The custom images are defined in the case config file and stored in the database. Scout currently supports `gif`, `jpeg`, `png` and `svg` images.
+
+One or more images constitute a group and are rendered together in a accordion-type UI elemment named after the group. The fields `title`, `description` and `path` are mandatory. The image size can be defined with the optional parameters `widht` and `height`. *Note*: adding images larger than 16mb are not reccomended as it might degrade the performance.
```

Comment:
I believe it was something that came out of the GMS meetings.

Since the images could be vital for the diagnose I think it makes sense storing them in the database along with the other vital information such as the called variants. By just storing paths, especially without accompanied hashes, the files them selves can change or disappear making difficult to go back to older cases.

Reply:
Fair enough.